huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_pg_plugin_test.go

```diff
@@ -0,0 +1,105 @@
+package rds
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getPgPluginResourceFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := cfg.NewServiceClient("rds", acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating PG plugin Client: %s", err)
+	}
+
+	reqURI := "v3/{project_id}/instances/{instance_id}/extensions?database_name={database_name}"
+	reqPath := client.Endpoint + reqURI
+	reqPath = strings.ReplaceAll(reqPath, "{project_id}", client.ProjectID)
+	reqPath = strings.ReplaceAll(reqPath, "{instance_id}", state.Primary.Attributes["instance_id"])
+	reqPath = strings.ReplaceAll(reqPath, "{database_name}", state.Primary.Attributes["database_name"])
+
+	resp, err := client.Request("GET", reqPath, &golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving PG plugin: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("error retrieving RDS PostgreSQL plugin: %s", err)
```
and fix other places